bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -0,0 +1,118 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button, fxa_cta_link with context %}
+
+{% add_lang_files "firefox/whatsnew_71" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - More privacy, more protections.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_71') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew71' %}
+{% set _utm_campaign = 'whatsnew71' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-emphasis-box">
+        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('No account required. But you might want one.') }}
```

Comment:
q: we opted to not do an alternative headline for users who are already signed in? (not a blocker.)

Reply:
_side-note, outside the scope of this PR:_
Perhaps we could consider collaborating with the fxa team to create a landing page experience for users who are already signed in who click a `fxa_cta_link`. Instead of being directed to sign in again, we could show the other products you can link to your account (monitor, lockwise, .etc.)

Reply:
Yeah this quetion has been asked. MEM approved only a single CTA for this page (no conditional states)